extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/VertxHttpProcessor.java

```diff
@@ -76,11 +75,9 @@ void filterMultipleVertxInstancesWarning(LaunchModeBuildItem launchModeBuildItem
     }
 
     @BuildStep(onlyIf = IsNormal.class)
-    @Record(value = ExecutionTime.RUNTIME_INIT, optional = true)
-    public KubernetesPortBuildItem kubernetes(HttpConfiguration config, VertxHttpRecorder recorder) {
-        int port = ConfigProvider.getConfig().getValue("quarkus.http.port", OptionalInt.class).orElse(8080);
-        recorder.warnIfPortChanged(config, port);
-        return new KubernetesPortBuildItem(config.port, "http");
+    public KubernetesPortBuildItem kubernetes() {
+        int port = ConfigProvider.getConfig().getOptionalValue("quarkus.http.port", Integer.class).orElse(8080);
```

Comment:
This is really wrong; it's a run time configuration property.  I think we need to revisit this.

Reply:
I'll think of a better solution for 1.2. Unfortunately without this change, Kubernetes manifest generation for 1.1 was sort of broken

Reply:
@dmlloyd yeah, we agreed with @geoand that there was something fishy. This is just a quick fix for 1.1.1. We need to discuss how we can solve this properly.